testservice_spaces.go

```diff
@@ -54,10 +55,11 @@ func spacesHandler(server *TestServer, w http.ResponseWriter, r *http.Request) {
 		}
 
 		if r.URL.Path == spacesURL {
-			var spaces []Space
+			var spaces []*Space
 			for i := uint(1); i < server.nextSpace; i++ {
```

Comment:
This seems surprising - why not `for spaceID, space := range server.spaces { .. }` ?